apps/server/src/controllers/ontimeController.ts

```diff
@@ -204,6 +204,7 @@ export const postSettings = async (req, res) => {
     const settings = DataProvider.getSettings();
     const editorKey = extractPin(req.body?.editorKey, settings.editorKey);
     const operatorKey = extractPin(req.body?.operatorKey, settings.operatorKey);
+    const serverPort = extractPin(req.body?.serverPort, settings.serverPort.toString());
```

Comment:
I am not sure we want to get the value from extractPin, this can never be null

We would need to add this value to the validation, so that, if it exists, it must be a numeric value
then you can just convert it using `Number(body.serverPort)`

Reply:
> I am not sure we want to get the value from extractPin, this can never be null

Well I just followed the pattern.

> We would need to add this value to the validation

Yeah, I will do that too.